README.md

```diff
@@ -40,6 +18,20 @@ You can use the [GOV.UK Docker environment](https://github.com/alphagov/govuk-do
 
 **Use GOV.UK Docker to run any commands that follow.**
 
+### Building the schemas
+
+```
+bundle exec rake build
+```
+
+### Running the tests
+
+```
+bundle exec rake spec
+```
+
+### Further documentation
+
 * [How to change a schema](docs/changing-a-schema.md)
```

Comment:
Very optional: this file and the one below seem to overlap with the "Suggested workflows" one (it's actually quite confusing which one to read first). We could just leave them as-is, but I thought I'd mention in case you feel inclined to merge them together.